course6/toyapp-adam/math/vector.h

```diff
@@ -42,6 +71,28 @@ struct Vector3D
 		z *= scalar;
 		return *this;
 	}
+
+	Vector3D Normalized() const
+	{
+		auto inverseLength = 1.0f / Length();
```

Comment:
what if Length is 0?

Reply:
Good question. Probably would be best to return the vector unchanged, imho.